README.md

```diff
@@ -2,15 +2,36 @@
 
 Implements fields and codes that are specific to European law.
 
-For complete guidance on meeting the disclosure requirements of European law, see [OCDS for European Union](https://standard.open-contracting.org/profiles/eu/master/en/).
+For complete guidance on meeting the disclosure requirements of European law, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/) and [OCDS for European Union](https://standard.open-contracting.org/profiles/eu/master/en/).
 
 ## Guidance
 
 If items have at most one delivery address, use the [Location](https://extensions.open-contracting.org/en/extensions/location/) extension instead ([see discussion](https://github.com/open-contracting/ocds-extensions/issues/115)).
 
 ## Legal context
```

Comment:
Maybe delete this section, because the EU extension exists to fill in gaps for eForms/TED that don't make sense to split out into other extensions (and it links to the profiles in the intro).

Reply:
Done in 382b0f56c80e342570a5bca220e543c7dfd2acc6